src/RoleObjects/GridData.js

```diff
@@ -380,19 +405,23 @@ export default class GridData extends TableData {
         cellChildIndex: -1,
       };
     } else {
-      _.forEach(cellDisplayObject.accessible.children, (cellChildDisplayObject, cellChildIndex) => { // eslint-disable-line max-len
-        if (cellChildDisplayObject.accessible.domId === id) {
-          matchingData = {
-            displayObject: cellChildDisplayObject,
-            sectionIndex,
-            rowIndex,
-            colIndex,
-            cellChildIndex,
-          };
+      _.forEach(
+        cellDisplayObject.accessible.children,
+        (cellChildDisplayObject, cellChildIndex) => {
+          // eslint-disable-line max-len
```

Comment:
```suggestion
        (cellChildDisplayObject, cellChildIndex) => {
```
shouldn't need this exception now